redirects.yaml

```diff
@@ -1065,3 +1063,34 @@ pricing/desktops: "/pricing/desktop"
 server/docs/virtualization-introduction: /server/docs/introduction-to-virtualization
 
 wsl/?: "/desktop/wsl"
+
+# Moved robotics docs
+robotics/docs/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/"
+robotics/docs/tutorials/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/"
+robotics/docs/ros-deployment-with-snaps-part-1/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/packaging-ros-application-as-snap/"
+robotics/docs/ros-deployment-with-snaps-part-2/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/packaging-complex-robotics-software-with-snaps/"
+robotics/docs/ros-deployment-with-snaps-part-3/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/distribute-ros-apps-with-snap-store/"
+robotics/docs/ros-deployment-with-snaps-part-4/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/building-ros-snaps-with-content-sharing/"
+robotics/docs/ros-deployment-with-snaps-part-5/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/create-ubuntu-core-image-for-turtlebot3/"
+robotics/docs/tutorial-ros-esm/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/tutorials/ubuntu-pro/ros-esm-intro/"
+robotics/docs/how-to-guides/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/how-to-guides/"
+robotics/docs/build-and-publish-a-ros-snap-with-github-actions/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/how-to-guides/packaging/build-and-publish-snap-with-github-actions/"
+robotics/docs/migrate-from-docker-to-snap/?: "https://canonical-robotics.readthedocs-hosted.com/en/latest/how-to-guides/packaging/migrate_from_docker_to_snap/"
```

Comment:
Just to understand why this change was introduced later and I see it's just not domain but slugs are changing as well, is there a reference doc to ensure that slugs are correct and we are not missing out any links.

Reply:
It is in the description of the pr:
- Sets up 121 redirects based on [this spreadsheet](https://docs.google.com/spreadsheets/d/1gsv-B0VoW-sz_N_wOfOsh7jZahaKUM-59PmvAHziSyI/edit?gid=0#gid=0)

Reply:
Got the access, the redirects seem good. I am approving PR.
Just to know more on "?" at end of url, I tested that it is able to match headings, what else can it match for?